lib/src/connector/timings.cc

```diff
@@ -203,6 +206,19 @@ AssociationTimings::Duration_min AssociationTimings::getOverallSessionInterval()
             boost::chrono::high_resolution_clock::now() - association);
 }
 
+AssociationTimings::Duration_ms AssociationTimings::getOverallSessionInterval_ms() const
```

Comment:
These are so similar you could pass the duration increment as a template argument. But this works fine.